tools/deb/istio-auth-node-agent.service

```diff
@@ -3,7 +3,7 @@ Description=istio-auth-node-agent: The Istio auth node agent
 Documentation=http://istio.io/
 
 [Service]
-ExecStart=/usr/local/bin/istio-auth-node-agent --logtostderr
+ExecStart=/usr/local/bin/node-agent --logtostderr
```

Comment:
that's a very generic name that shouldn't be as is in /usr/local/bin specially in a VM context

Reply:
Why is the name bing changed ?

Reply:
apparently it's not trivial to rename in bazel (!) so in the meantime, I think it's cleaner anyway to have that file in an istio/ named directory : proposing #270